src/models/sensor-gdx-manager.ts

```diff
@@ -0,0 +1,220 @@
+import { SensorConfiguration } from "./sensor-configuration";
+import { SensorManager, NewSensorData } from "./sensor-manager";
+import { SensorConfig } from "@concord-consortium/sensor-connector-interface";
+import godirect from "@vernier/godirect"
+
+export class SensorGDXManager extends SensorManager {
+    supportsDualCollection = true;
+
+    private internalConfig: SensorConfig;
+    private hasData: boolean = false;
+    private stopRequested: boolean = false;
+    private disconnectRequested: boolean = false;
+    private gdxDevice: any;
+    private enabledSensors: any;
+    private initialColumnNum = 100;
```

Comment:
Is there a methodology behind the column numbering?  I started at 100 in line with the sensor tag implementation, but this felt arbitrary.  Obviously columns need to be distinct when using a device with multiple sensors/sensor channels, but are there other patterns to the numbering?